packages/storage-documents/src/page-scan-run-result-map.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export declare type DocumentType = 'pageScanRunResult' | 'pageScanRunSarifReport';
+
+/**
+ * The mapping document between scan run ID and all scan run corresponding documents.
+ */
+export interface PageScanRunResultMap {
```

Comment:
how is this different from PageScanRunResult? what adidtional info this has?

Reply:
This is mapping file located under lookup folder with links pointing to historical folder of an actual scan results